Tests/TuistKitAcceptanceTests/AccountAcceptanceTests.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+import ServiceContextModule
+import TuistAcceptanceTesting
+import TuistSupport
+import TuistSupportTesting
+import XCTest
+
+@testable import TuistKit
+@testable import TuistServer
```

Comment:
```suggestion
import TuistServer
```

it's good practice to mark only one module to be `@testable`, usually should be the one that your test module is prefixed with (in this case `TuistKit`). Is there a need to mark `TuistServer` as `@testable`, too?